### PR TITLE
feat(example): add hello-sparkles example

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

File: examples/templates/hello-world.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
+  <body>
+    <h1>Hello {{ name }}!</h1>
+  </body>
+</html>
+
```
Comment (steveklabnik) [inline examples/templates/hello-world.html]: ✂️

File: examples/templates/hello-world.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
```
Comment (steveklabnik) [inline examples/templates/hello-world.html]: > Error: Element head is missing a required instance of child element title.

**Review by steveklabnik — CHANGES_REQUESTED** [review]

File: examples/templates/hello-world.html (hunk 1 above)
Comment (ashleygwilliams) [reply examples/templates/hello-world.html]: eee whoops

File: examples/templates/hello-world.html (hunk 2 above)
Comment (ashleygwilliams) [reply examples/templates/hello-world.html]: ok fiiiine haha

**Review by steveklabnik — APPROVED** [review]

**Comment by steveklabnik** [comment]
:metal: ✨ 💖

**Pull request merged**